fish-pepper/run-java-sh/fp-files/run-java.sh

```diff
@@ -462,7 +462,7 @@ proxy_options() {
 
   local noProxy="${no_proxy:-${NO_PROXY:-}}"
   if [ -n "$noProxy" ] ; then
-    ret="$ret -Dhttp.nonProxyHosts=\"$(echo $noProxy | sed -e 's/,[[:space:]]*/|/g')\""
+    ret="$ret -Dhttp.nonProxyHosts=\"$(echo $noProxy | sed -e 's/,[[:space:]]*/|/g' | sed -e 's/|\./|\*\./g')\""
```

Comment:
what if the first part starts with a `.` ? E.g. what's about `.example.com, localhost` ? Then the second sed call want match and you don't get the proper replacement.

Instead I'd suggest something like

```
echo "|$noProxy" | sed -e 's/,[[:space:]]*/|/g' | sed -e 's/|\./|\*\./g' | cut -c 2-
```

I would also add a corresponding test for ".example.com, localhost"